internal/api/core/route.go

```diff
@@ -106,5 +111,28 @@ func MustNewAPIRoute(basePath string, apiHandler any) *APIRoute {
 		},
 		RequestType:  requestType,
 		ResponseType: responseType,
+		Description:  "",
 	}
+
+	absPath := makeAbsLocalAPIPath(basePath) + ".go"
+	fset := token.NewFileSet()
+	node, parseErr := parser.ParseFile(fset, absPath, nil, parser.ParseComments)
+	if parseErr != nil {
+		log.Warn("Error while parsing file to get API description.", parseErr, log.NewAttr("endpoint", absPath))
```

Comment:
Should this be an error?
Under what conditions can we fail a parse (given that we are running a binary that should have the compiled code in it)?